docs/guis/usage/releases.md

```diff
@@ -1,6 +1,105 @@
-# Software Releases
+# Releasing the Beta-Beating GUI
 
-## Releasing the GUIs
+!!! warning "GUI Releases"
+    This page describes the process of releasing a new version of the [Beta-Beating GUI](../betabeat/gui.md),
+    as this is the main Java GUI we are responsible for.
+    The python GUIs are released following the standard [python release process](../../packages/development/howto_release.md).
+    The release of the other Java GUIs should be similar to this guide, but please contact the respective maintainer of the GUI for further details.
 
-!!! todo
-    Describe how to release the GUIs
+## Normal Release
+
+!!! tip "Preparation"
+    - Add new features to the [changelog][bbgui_changelog]{target=_blank .cern_login}:<br>
+      To track changes and easily identify breaking changes, please add them to the [CHANGELOG.md][bbgui_changelog]{target=_blank .cern_login}.
+      Preferrably, this should be done already before merging the changes into the master branch, but should be at least be ensured to be **up-to-date during release**.
+    - Update version in [product.xml][bbgui_product_xml]{target=_blank .cern_login}:<br>
+      The version number needs to be updated in the [product.xml][bbgui_product_xml]{target=_blank .cern_login} file.
+      **If an already used version number/identifyer is used, the release procedure will fail.**
+    - Have a clean repository and no uncommitted changes:<br>
+      The release process runs in the current folder of your local machine and will fail if there are any uncommitted changes in the repository.
+      **Make sure your repository is synced and up-to-date.**
+
+!!! success "How to run"
+    To run the task for a **normal release**, start CBNG's `release` from the GUI (e.g. in VSCode in `Gradle > Gradle Projects > betabeating-app > Tasks > cbng > release`) or run
+
+    ```bash
+    bob release
+    ```
+
+    from commandline.
+
+This task will build the project, run tests (if they existed, see admonition below) and automatically upload the `.jnlp` file to the
+[be deployment servers][deployment_url] at:
```

Comment:
I think you can capitalize BE for clarity. Took me a second (but I lack coffee sooo, your call)